### DISCUSSION TITLE
Typescript error for zustand 4.4.5 together with immer 10.0.3

### DISCUSSION THREAD

**Discussion by chessydk** [pmndrs/zustand#2169] [Bug report] [answered]
When updating to 4.4.5 I get a typescript error when using immer as middleware:

![image](https://github.com/pmndrs/zustand/assets/1286704/83f658ac-edf6-471c-a9c4-f2fbadc9fa73)

**Comment by dai-shi** [comment, marked as answer]
See also #2163
